Modules/Module_Processing/xASL_init_LoadDataPar.m

```diff
@@ -134,4 +136,61 @@
     %% Load dataPar
     x = xASL_adm_MergeStructs(dataPar.x, x);
     
-end
\ No newline at end of file
+end
+
+%% -----------------------------------------------------------------------------
+%% -----------------------------------------------------------------------------
+function [x] = xASL_initLoadDataPar_PrepareAtlas4ROI(x)
```

Comment:
Shouldn't we keep/move this to the population module where it belongs (same as the check above)?

As it (should have) no effect on the rest of the processing.
But on the other hand, perhaps good to issue warnings here rather than at the end for users that specify this in dataPar.json

Reply:
We had it in the population module. And you asked to move this to the dataPar loading ;)

So I'd stick with the original comment that we want to warn the user about a basic wrong input right at the start.